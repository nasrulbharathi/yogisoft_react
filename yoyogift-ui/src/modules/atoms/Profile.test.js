import React from "react";
import { shallow } from "enzyme";
import Profile from "./Profile";

describe("Profile", () => {
  it("should be defined", () => {
    expect(Profile).toBeDefined();
  });
  it("should render Profile", () => {
    const props = {
      detailsObject: {
        email: "nasrulrules@gmail.com",
        firstName: "nasrul",
        lastname: "bharathi",
        socialProfileLink: "http",
        picture: "http://img.png",
        balance_points: 200,
      },
    };
    const component = shallow(<Profile {...props} />);
    expect(component).toMatchSnapshot();
  });
});
