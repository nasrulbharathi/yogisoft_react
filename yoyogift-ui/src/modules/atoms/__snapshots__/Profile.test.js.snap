// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should render Profile 1`] = `ShallowWrapper {}`;
